composer.json

```diff
@@ -20,8 +20,8 @@
 
 	"require-dev": {
 		"satooshi/php-coveralls": "^1.0",
-		"phpunit/php-file-iterator": "1.3.3",
-		"phpunit/phpunit": "~4.0"
+		"phpunit/php-file-iterator": "^1.4",
+		"phpunit/phpunit": "^4.8.35"
```

Comment:
Why not changing this to something that allows all major PHPUnit versions?

Reply:
`PHPUnit 5` requires `PHP 5.6`. I guess we need to stay in `PHPUnit 4`.

Reply:
I'm saying to allow all PHPUnit versions, not just PHPUnit 6. This way during tests on Travis CI on each PHP version most suitable PHPunit version will be used.

Constrain like `>=4.8.35 <5|>=5.4.3 <6` would be perfectly fine I guess.